src/encoded/static/components/antibody.js

```diff
@@ -105,11 +105,13 @@ const LotComponent = (props, reactContext) => {
         { id: geneComponents.length > 0 ? geneComponents : null, query: geneQuery, tip: geneTips.join(' + ') },
     ];
 
+    const crumbsReleased = (context.status === "released");
```

Comment:
This will apply to almost every breadcrumb change: strings (but not HTML attribute values) have to use single quotes: https://github.com/airbnb/javascript#strings